src/js/actions/tools.js

```diff
@@ -446,10 +464,16 @@ define(function (require, exports) {
     };
     selectTool.reads = [];
     selectTool.writes = [locks.JS_TOOL, locks.PS_TOOL];
-    selectTool.transfers = [installShapeDefaults, shortcuts.addShortcut,
-        shortcuts.removeShortcut, "layers.deleteSelected", "layers.resetLayers",
-        policy.removePointerPolicies, policy.removeKeyboardPolicies, policy.addPointerPolicies,
-        policy.addKeyboardPolicies, policy.setMode];
+    selectTool.transfers = [swapPolicies, policy.removePointerPolicies,
```

Comment:
Each of these various select/deselect transfers has roughly the same scope, so other than being an "ugly long list" this doesn't really have much down side.  eh?